### PR TITLE
[RDAPI-27669] Changes to Transaction Status filter in Traffic Monitor

### PR DESCRIPTION
**Pull request by Bcolemanaxway**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Note to describe the new behaviour of the Transaction Status filter when set to Block.

Also a clean up of dead content on the "Monitoring and metrics" page

## Deploy preview link

http://pr-2470.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_administration/apigtw_admin/monitor_service/

http://pr-2470.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_relnotes/20221130_apimgr_relnotes/

## Checklist for contributors

Before submitting this PR, please make sure:

_This is simply a reminder of what we are going to look for before merging your change._

* [X] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [X] You have signed the [Axway CLA](https://cla.axway.com/)
* [X] You have verified the technical accuracy of your change
* [X] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [X] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [X] You have verified that all status checks have passed

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 92f047e |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-2470.axway-open-docs.opendocs-builder.pcloud.axway.int |

**Comment by andreamussap** [comment]
Ensure that the following images, which are not referenced anymore, aren't used somewhere else, then remove them from the repository:

/Images/APIGateway/rt_monitoring.gif
/Images/APIGateway/message_monitoring.png
/Images/APIGateway/message_content.gif
/Images/APIGateway/message_traffic_perf.gif
/Images/APIGateway/blocked_message_path.gif

Done! See commit - https://github.com/Axway/axway-open-docs/pull/2470/commits/52f4523911037d9c179887641c9a9437092b5fc4

File: content/en/docs/apim_administration/apigtw_admin/monitor_service.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ This section explains how to monitor example services using the API Gateway Mana
 You can view a wide range of monitoring data in the API Gateway Manager. For example, this includes message status, message traffic, filter execution path, message content, system, services, and remote hosts. You can view real-time traffic monitoring summary data on the main **Dashboard**
 tab in the **TRAFFIC** section. The following example shows the number of messages that have been passed by the API Gateway on to a service:
 
-![Real-time monitoring](/Images/APIGateway/rt_monitoring.gif)
+![Real-time monitoring](/Images/APIGateway/passed_widget.png)
```
[the remark this answers is not in the document]
Comment (Bcolemanaxway) [reply content/en/docs/apim_administration/apigtw_admin/monitor_service.md:30]: Apologies! Adding now.

**Comment by Bcolemanaxway** [comment]
> 

Thanks @andreamussap !

**Review by scormicanAxway — APPROVED** [review]

**Review by JohnD-Axway — APPROVED** [review]

**Pull request merged**